src/MakeJsonCommand.php

```diff
@@ -240,12 +239,18 @@ protected function make_json( $source_file, $destination, $map ) {
 
 		$base_file_name = basename( $source_file, '.po' );
 
-		foreach ( $translations as $index => $translation ) {
+		$domain = $translations->getDomain();
+
+		$base_file_name_with_domain = $domain && 0 !== strpos( $base_file_name, $domain ) ?
```

Comment:
This needs parentheses to denote the intention clearly.
```suggestion
		$base_file_name_with_domain = ( $domain && 0 !== strpos( $base_file_name, $domain ) ) ?
```

Reply:
It was an IDE recommendation to remove unneeded parentheses.

Happy to add them back, since I shouldn't have included unrelated changes here anyway.

Is there perhaps a lint rule that could enforce this kinda style?

Reply:
The parentheses are technically not needed because of the precedence rules of the different operators.

However, given this specific construct, I had a hard time reading it correctly in its raw form. I had initially thought there was a bug with an inadvertent assignment.

Reply:
Rewrote it now without a ternary